src/main/java/com/sap/sgs/phosphor/fosstars/model/value/ScoreValue.java

```diff
@@ -80,12 +80,16 @@ public ScoreValue(
     this.usedValues = new ArrayList<>(Objects.requireNonNull(usedValues, "Values can't be null!"));
   }
 
-  @Override
   @JsonGetter("score")
-  public Score feature() {
+  public Score score() {
```

Comment:
Would this be more appropriate if the method name is value()?

Reply:
This method doesn't return the value. It returns a score which calculated the value.